---a/net/minecraft/network/play/server/S1CPacketEntityMetadata.java
+++b/net/minecraft/network/play/server/S1CPacketEntityMetadata.java
@@ -2,56 +2,49 @@
 
 import java.io.IOException;
 import java.util.List;
+
 import net.minecraft.entity.DataWatcher;
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S1CPacketEntityMetadata implements Packet
-{
+
+public class S1CPacketEntityMetadata implements Packet {
+
     private int field_149379_a;
     private List field_149378_b;
     private static final String __OBFID = "CL_00001326";
 
-    public S1CPacketEntityMetadata() {}
+    public S1CPacketEntityMetadata() {
+    }
 
-    public S1CPacketEntityMetadata(int p_i45217_1_, DataWatcher p_i45217_2_, boolean p_i45217_3_)
-    {
+    public S1CPacketEntityMetadata(int p_i45217_1_, DataWatcher p_i45217_2_, boolean p_i45217_3_) {
         this.field_149379_a = p_i45217_1_;
 
-        if (p_i45217_3_)
-        {
+        if (p_i45217_3_) {
             this.field_149378_b = p_i45217_2_.getAllWatched();
-        }
-        else
-        {
+        } else {
             this.field_149378_b = p_i45217_2_.getChanged();
         }
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.field_149379_a = buf.readVarIntFromBuffer();
         this.field_149378_b = DataWatcher.readWatchedListFromPacketBuffer(buf);
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeVarIntToBuffer(this.field_149379_a);
         DataWatcher.writeWatchedListToPacketBuffer(this.field_149378_b, buf);
     }
 
-    public void func_180748_a(INetHandlerPlayClient p_180748_1_)
-    {
+    public void func_180748_a(INetHandlerPlayClient p_180748_1_) {
         p_180748_1_.handleEntityMetadata(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public List func_149376_c()
-    {
-        return this.field_149378_b;
+    public void processPacket(INetHandler handler) {
+        this.func_180748_a((INetHandlerPlayClient) handler);
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.func_180748_a((INetHandlerPlayClient)handler);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public int func_149375_d()
-    {
-        return this.field_149379_a;
-    }
-}
+}
